commit 105f20486c171a3858172e792d341950106999bd
Author: leo.singer@LIGO.ORG <leo.singer@LIGO.ORG@ac820499-f4a4-4ea9-b4e2-a7172f5148fc>
Date:   Wed Sep 11 21:05:20 2013 +0000

    Drop Jikes because it can't compile Java 1.5 source
    
    Also make build GCJ-compatible while we are at it.
    
    git-svn-id: https://svn.ligo.caltech.edu/svn/nds2/trunk@1154 ac820499-f4a4-4ea9-b4e2-a7172f5148fc

diff --git configure.ac configure.ac
index 6be4af8..e5c388f 100644
--- configure.ac
+++ configure.ac
@@ -136,7 +136,7 @@ dnl Numpy
 AX_PYTHON_MODULE([numpy])
 
 dnl javac
-AC_CHECK_PROGS([JAVAC], ["gcj -C" guavac jikes javac])
+AC_CHECK_PROGS([JAVAC], ["gcj -C -fsource=1.5 -ftarget=1.5" "javac -source 1.5 -target 1.5"])
 AC_ARG_VAR([JAVAC], [path to javac tool])
 
 dnl docbook
diff --git swig/java/Makefile.am swig/java/Makefile.am
index 64df5a4..f5d022c 100644
--- swig/java/Makefile.am
+++ swig/java/Makefile.am
@@ -7,8 +7,6 @@ swig_sources = $(top_srcdir)/swig/nds2.i $(srcdir)/nds2_java.i $(srcdir)/jniload
 java_classes=$(built_java_sources:%.java=nds2/%.class)
 CLEANFILES = $(built_sources) nds2Constants.java nds2.java
 
-# Options for javac to generate classes that are compatible with ancient JREs
-AM_JAVACFLAGS = -source 1.5 -target 1.5
 JAVAROOT = $(builddir)
 
 if COND_SWIG_JAVA
